src/cephadm/cephadm

```diff
@@ -2756,17 +2760,26 @@ def get_container(ctx: CephadmContext,
 
     # if using podman, set -d, --conmon-pidfile & --cidfile flags
     # so service can have Type=Forking
+    # set containers limits to unlimited as defaults (Docker 4096 / Podman 2048)
+    # prevents some app customizations from running
     if isinstance(ctx.container_engine, Podman):
+        pids_unlimited = '-1'
         runtime_dir = '/run'
         container_args.extend([
             '-d', '--log-driver', 'journald',
             '--conmon-pidfile',
             runtime_dir + '/ceph-%s@%s.%s.service-pid' % (fsid, daemon_type, daemon_id),
             '--cidfile',
             runtime_dir + '/ceph-%s@%s.%s.service-cid' % (fsid, daemon_type, daemon_id),
+            '--pids-limit=%s' % (pids_unlimited),
```

Comment:
Nit: is `pids_unlimited` variable actually needed here?
```suggestion
            '--pids-limit', '-1',
```

Reply:
Not really. I will remove it.